src/main/java/org/fundacionjala/sfdc/pageobjects/HomeBasePage.java

```diff
@@ -0,0 +1,9 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.pageobjects.components.TopMenu;
+
+public abstract class HomeBasePage extends BasePage {
+    public TopMenu topMenu;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make topMenu a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)